### PR TITLE
Authpoc

### PR DESCRIPTION
**Pull request by marcusellison**
- [x] integrate user info
- [x] update design and fix conflicts
- [x] change redirect urls
- [x] add user initials
- [x] add private routes

### CODE REVIEW

File: src/component/PostDetails2.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  // 2. Contents for slots you want to fill,
+  // 3. Overrides for any named node in the component to attach behavior and data,
+  // 4. Props to set on the root node.
+  //
+  // By default, we are just piping all PostDetails2Props here, but feel free
+  // to do whatever works for you.
+  return <PlasmicPostDetails2 {...props} />;
+}
+
+export default PostDetails2;
```
Comment (xh3b4sd) [inline src/component/PostDetails2.tsx:42]: This doesn't look right.

File: src/index.tsx [hunk 2]
```diff
@@ -4,21 +4,42 @@ import { QueryClient, QueryClientProvider } from "react-query";
...
+  history.push(
+    appState && appState.returnTo ? appState.returnTo : window.location.pathname
+  );
+};
+
+const providerConfig = {
+  domain: config.domain,
+  clientId: config.clientId,
+  ...(config.audience ? { audience: config.audience } : null),
+  redirectUri: "http://localhost:3006/profile",
```
Comment (xh3b4sd) [inline src/index.tsx:29]: I think we should follow the configuration mechanism we already have. Let's not hard code magic strings like that.

**Review by xh3b4sd — COMMENTED** [review]
I was looking for two options here, but I am not sure you want to address all of them here. 

- Use web workers instead of cookies and local storage. 
- Use refresh tokens.

**Comment by marcusellison** [comment]
> I was looking for two options here, but I am not sure you want to address all of them here.
> 
> * Use web workers instead of cookies and local storage.
> * Use refresh tokens.

My understanding from our conversation with Peter was that cookies are not a problem (depending on implementation) and that refresh tokens were things we don't have to worry about because it's managed in the SDK.

**Comment by xh3b4sd** [comment]
We do not have to worry about it if we configure it. 

Screenshot 2021-02-20 at 23 05 09

See https://auth0.com/docs/quickstart/spa/react/02-calling-an-api.

**Pull request merged**